packages/app/src/explorer/EntityItem.tsx

```diff
@@ -55,6 +57,37 @@ function EntityItem(props: Props) {
 
           onSelect(path);
         }}
+        onKeyDown={(e) => {
+          switch (e.key) {
+            case 'ArrowDown':
+              focusNext(e);
+              return;
+
+            case 'ArrowUp':
+              focusPrevious(e);
+              return;
+
+            case 'ArrowLeft':
+              if (isGroup(entity) && isExpanded) {
+                toggleExpanded(false);
```

Comment:
I wonder if we should keep a `toggle` here given it is more used as a `set` now.

Reply:
I like the toggle, it makes it clear it's a boolean flag.